experiments/amg2023/experiment.py

```diff
@@ -54,84 +54,57 @@ class Amg2023(
     #     description="value of n",
     # )
 
-    def compute_applications_section(self):
-        # TODO: Replace with conflicts clause
-        scaling_modes = {
-            "strong": self.spec.satisfies("+strong"),
-            "weak": self.spec.satisfies("+weak"),
-            "throughput": self.spec.satisfies("+throughput"),
-            "single_node": self.spec.satisfies("+single_node"),
+    def initialize_expr_variables(self):
```

Comment:
amg: number of zones is number of zones per process
strong scaling: 2 procs 64 x 64 x 64 -> on 4 process is 64 x 64 x 32 -> on 8 is 64 x 32 x 32
weak scaling: always 64 x 64 x 64